src/main/java/software/amazon/awssdk/crt/checksums/Checksums.java

```diff
@@ -18,6 +19,24 @@ public static int crc32c(byte[] input) {
         return crc32c(input, 0, 0, input.length);
     }
 
+    public static int crc32(int input, int prev) {
```

Comment:
What are these for?

Reply:
Java team asked that we add functionality to pass in ints so that they have an easier time matching the java checksum interface, and this was easier than arguing.

Reply:
I don't think this is matching what they're asking for though.  The update is for bytes not ints.  And honestly let's just implement the checksum interface so that they (and anyone else who ever uses this) don't have to do it themselves.